### ISSUE TITLE
Ability to select and close conversations in bulk

### ISSUE THREAD

**Issue by Pnlpz** [chaskiq/chaskiq#647]
When you have the integration with Slack, you rarely go to the Chaskiq app to check or close conversations. And when you do, there are so many conversations to be closed that is overwhelming. 

It would be great to have either a button within Slack to close each conversation OR the ability to select and close conversations in bulk. 

I recall you had set a condition that closed conversations after a period of time(?), correct me if I'm wrong. If that is so, after how much time do conversations are closed? 🤔

Thanks! <3

**Comment by michelson** [comment]
This could be a good addition, thanks for the suggestion. But did you know
that you can close conversations directly from the slack app right?

**Comment by Pnlpz** [comment]
You are right! I did not see that button. Now, when I click it, the thread
is closed and all the messages are displayed in the channel (it may be a
little bit noisy, but it's a great solution anyway).
Thanks @michelson !

**Comment by michelson** [comment]
Hey @Pnlpz what you mean with: 
> Now, when I click it, the thread
is closed and all the messages are displayed in the channel
